gokart/target.py

```diff
@@ -24,20 +26,23 @@ def exists(self) -> bool:
         return self._exists()
 
     def load(self) -> Any:
-        return self._load()
+        return self._with_lock(self._load)()
 
     def dump(self, obj) -> None:
-        self._dump(obj)
+        self._with_lock(self._dump)(obj)
 
     def remove(self) -> None:
-        self._remove()
+        return self._with_lock(self._remove)()
 
     def last_modification_time(self) -> datetime:
         return self._last_modification_time()
 
     def path(self) -> str:
         return self._path()
 
+    def _with_lock(self, func):
+        return with_lock(func=func, redis_params=self._redis_params)
```

Comment:
@mski-iksm [IMO] what happens when self._redis_params is not set? Plz care that situation :)

Reply:
Maybe default param of `self._redis_params` should be set as None?

Reply:
👍

Reply:
@Hi-king 
`self._redis_params` must be an instance of `RedisParams`.

To care such cases, I'll set a default value at  `self._redis_params`.